packages/payment-detection/test/any/any-to-erc20-proxy-contract.test.ts

```diff
@@ -98,6 +119,27 @@ describe('api/any/conversion-fee-proxy-contract', () => {
     });
   });
 
+  it('can createExtensionsDataForCreation with fee amount and address (Goerli)', async () => {
```

Comment:
Same.
i.e:  testSuite(network: string) ... you can find an example in requestNetwork/packages/payment-processor/test/payment/erc20-batch-proxy.test.ts